tests/testthat/test-duplicates.R

```diff
@@ -0,0 +1,14 @@
+test_that("can read multiple hobo csv file", {
```

Comment:
This test is not well named, I think it may be an old copy paste error. It would be good to add a few more tests as well as there is not a lot of tests.

Reply:
Let me know if you'd like me to add any more tests